src/logic/utils.ts

```diff
@@ -1,14 +1,28 @@
 import {
     NEW_PLAYER_HIGHLIGHT_DAYS,
+    NUMBER_OF_PLAYERS_FOR_VALID_MATCH,
+    PLAYER_MAXIMUM_GAMES_AS_NEW_PLAYER,
     PLAYER_MINIMUM_GAMES_REQUIRED,
     PLAYER_MINIMUM_WINS_REQUIRED
 } from "../components/constants";
 import { Player } from "../types/domain/Player";
 
+
+/**
+ * Gets the win rate as a percentage (number, without the "%").
+ * @param winCount Wins - most likely valid wins.
+ * @param totalCount Number of matches to consider - most likely valid matches.
+ * @returns A whole number from 0 to 100.
+ */
 export function getWinRatePercentage(winCount: number, totalCount: number) {
     return totalCount > 0 ? Math.round((winCount / totalCount) * 100) : 0;
 }
 
+/**
+ * Gets the average win turn of a player.
+ * @param player Has to be type Player. Filters out automatically matches that don't meet the player count requirement.
```

Comment:
nit: Move the second sentence to the description of the function. 

meanwhile, the description of the parameter can be sometihng like:
@param player The player to determine average win turn off of. 

super boring, but very accurately describing what the parameter represents

Reply:
woopsie! ty for catching that lol